CONTRIBUTING

```diff
@@ -0,0 +1,60 @@
+
+
+Desired PR workflow
+Info on setup / running tests
+Conventions / style preferences.
+
+## How to contribute
+We're excited for contributions to Great Expectations. If you see places where the code or documentation could be improved, please get involved!
+
+Submitting your changes
+Once your changes and tests are ready to submit for review:
+
+1. Test your changes
+
+    Run the test suite to make sure that nothing is broken. See the the section on testing below for help running tests. (Hint: `pytest` from the great_expectations root.)
```

Comment:
If pytest is the recommended / supported test runner and discovery tool, should it also be in requirements-dev?

Reply:
Good catch